msteams-platform/m365-apps/specify-runtime-requirements.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Specify App Runtime Requirements for Microsoft 365
+description: Specify app runtime requirements, such as one-way or mutual dependencies and capabilities, to tailor user experience in different Microsoft 365 hosts.
+ms.date: 7/10/2024
+ms.author: mosdevdocs
+author: erikadoyle
+ms.topic: conceptual
+ms.localizationpriority: medium
+ms.subservice: m365apps
+---
+# Specify Microsoft 365 host runtime requirements in your app manifest
+
+> [!NOTE]
+>
+> The ability to specify Microsoft 365 host runtime requirements in app manifest (previously called Teams app manifest) is in [public developer preview](../resources/schema/manifest-schema-dev-preview.md).
+
+When you upgrade your Microsoft Teams personal tab or message extension app to use app manifest version 1.13 or later, it's available in other Microsoft 365 application hosts by default. However, if your app also includes capabilities not yet supported in these hosts, your app might only partially load, resulting in unplanned user experiences.
```

Comment:
@erikadoyle, do we need to mention bots here as well? The feedback from reviews is that we haven't mentioned that this feature applies to bots as well and that's not mentioned anywhere at the beginning, only to be suddenly mentioned under `elementRelationshipSet`.

Reply:
How about this (taken from a real-world partner problem that RT will solve): 

For example, if your app is defined with app manifest schema version 1.17 and includes a bot and a personal tab that represents the bot's configuration settings, your app would load in Outlook and Microsoft 365 (Office) app, but only surface the bot configuration tab to the user without the bot itself.